setup/artifacts/s3.md

```diff
@@ -7,7 +7,7 @@ sidebar:
 
 {% include toc %}
 
-Spinnaker stages that read data from artifacts can read S3 files directly.
+Spinnaker stages that read data from artifacts can read S3 files directly. Setting `--api-endpoint` & `--api-region` is optional. It is needed only if you are using a S3 clone such as Minio. 
```

Comment:
What do you think about moving this to below the code snippet in  "2. Add an Artifact to your account"? 
We're introduced to these flags there, so it might flow better to make mention of them there like: "Note: setting `--api-endpoint` and..."

Reply:
Make sense. I'll move it there :)